domains/fun.json

```
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "wsmdn",
+           "email": "",
+           "discord": "1049856856467722242",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.QLkC6cgKHDBiJbqCaXbLiRRD4ahPjxUEoaQw80SoxKGFgZFAqc1yMOhSlsByLNqO_bAsuvMi2lcrhFJfNQywEUshUKLLEmSzP_LMmzEgXOHQF1iVe6Qvs0_S16znGryZBsZRuwiu7kKbCxhGR6Jv4ZmLsAKH8xGSXQ-B9FClhSXqCwYKDEJ3-zud-3_SlrBnEpiAtwz0qDlQPBhDT-LUAiHiyXps3gPDjqRgWJdJoya6czWJBK04f_R4dDr0OyNIM_xWTn6bMZVMJKSNmWIece93iVfigFD1xr4Gy8wgvdgjTsyQXHi8B_JNVpD3xRXCq4ZRGxzqic8_ScyoOfXsAg.rltw9Lf48PyxDAYcP20L-Q.FiaPAtdlS4I9mUTAbBmPh2gK_KTAhiX8xTk_eGZ8t9uaN7GLk2lnwhGfFEnIwm_A21khJB3DdNq1TNEsgjYvh8wLCnfQoQfQ5XZSXWKlans.JkoAoo5Tv9Gdckn4t-tAAQ"
+        },
+    
+        "record": {
+            "CNAME": "wsmdn.github.io/more/nibeipianle.html"
```

```suggestion
            "CNAME": "wsmdn.github.io"
```
CNAME cannot contain slashes `/`.